tests/test_py36.py

```diff
@@ -0,0 +1,300 @@
+"""
```

Comment:
looks like much of this might be duplicates of the python 3.7 tests? couldn't we just remove the "skip" and run the same test for both? (At least in some cases).

Reply:
Python 3.6 and pyhton3.7 are a little different (eg.  _ForwardRef and ForwardRef).
I have to change exists test code.
I try to do it.

Reply:
great, I think better to have one set of tests than two, even if it means a bit of repeated "try:...except:` or `if sys.version...else`.

Reply:
I have changed test_py37.py to test_forward_ref.py  and deleted test_py36.py